client/components/PlanningList/index.jsx

```diff
@@ -33,7 +34,8 @@ class PlanningList extends React.Component {
                     onUnspike={handlePlanningUnspike}
                     onClick={previewPlanning}
                     onDoubleClick={openPlanningEditor}
-                    privileges={privileges} />
+                    privileges={privileges}
+                    session={session}/>
```

Comment:
I'm not convinced that `<PlanningItem/>  needs to be aware of the `session`. Maybe just a `isLocked` boolean might be enough ?